src/sedcli_main.c

```diff
@@ -63,7 +63,7 @@ static cli_option reverttper_opts[] = {
 
 static cli_option lock_unlock_opts[] = {
 	{'d', "device", "Device node e.g. /dev/nvme0n1", 1, "DEVICE", CLI_OPTION_REQUIRED},
-	{'t', "locktype", "String specifying how to lock/unlock drive. Allowed values: RW/RO/WO/UNLOCK", 1, "FMT", CLI_OPTION_REQUIRED},
+	{'t', "accesstype", "String specifying access type to the drive. Allowed values: RO/RW/LK", 1, "FMT", CLI_OPTION_REQUIRED},
```

Comment:
Please change description to "String specifying access type to the data on drive. Allowed ..."